### DISCUSSION TITLE
Thanks for creating this repo!

### DISCUSSION THREAD

**Discussion by ZmiennyVT** [152334H/DL-Art-School#66] [Show and tell]
I wanted to write this post as a **thank you** to the author and everyone involved 😄 
Thanks to your work, both me and the community around TTS models have been able to create amazingly sounding voices in languages other than English! The base model and zero-shot cloning work great, and the fine-tuning is absolutely mind-blowing! :)

- http://sndup.net/znqs
- http://sndup.net/kqnc

It's really hard to find a good model for the Polish language and others, and finally, there's something that's incredibly useful, for example, in creating dubbing or reproducing voices of historical characters.
I'm waiting to see if it's possible to make tortoise-fast generate voices in real-time, for use in applications like virtual assistants. You can't even imagine how big of a shock it would be to have a smart, speaking Siri in some countries 😄 
Anyway, once again, thank you! And best regards

**Comment by serkandyck** [comment]
> I wanted to write this post as a **thank you** to the author and everyone involved 😄 Thanks to your work, both me and the community around TTS models have been able to create amazingly sounding voices in languages other than English! The base model and zero-shot cloning work great, and the fine-tuning is absolutely mind-blowing! :)
> 
> * http://sndup.net/znqs
> * http://sndup.net/kqnc
> 
> It's really hard to find a good model for the Polish language and others, and finally, there's something that's incredibly useful, for example, in creating dubbing or reproducing voices of historical characters. I'm waiting to see if it's possible to make tortoise-fast generate voices in real-time, for use in applications like virtual assistants. You can't even imagine how big of a shock it would be to have a smart, speaking Siri in some countries 😄 Anyway, once again, thank you! And best regards

It was a very good training, some robotic voices are heard, but I'm sure you can develop much better models, could you give some information about how you train with a language other than English?
